operations/boot_orchestration/Components.md

```diff
@@ -39,7 +39,7 @@ Stores information on the eventual desired boot artifacts and configuration for
 
 ### `enabled`
 
-If the node is enabled (enabled == True), BOS will take action to make the actual state match the desired state.
+If the node is enabled (enabled == True), BOS will take action to make the actual state match the desired state. Disabled nodes may receive status updates from booted nodes, but BOS will not issue power commands to the nodes while they are disabled.
```

Comment:
This may be obvious but might be worth saying "... while they are disabled (i.e. enabled == False)."

Reply:
I had a more verbose version before but changed it during peer review while the PR was in WIP.